### ISSUE TITLE
DISABLED test_accurracy (__main__.TrainMnist)

### ISSUE THREAD

**Issue by mrshenli** [pytorch/pytorch#34262]
Try if our chrono job also works for tests from other repo.

https://app.circleci.com/jobs/github/pytorch/pytorch/4693216

The failure is due to torchvision dataset downloads from `http://yann.lecun.com/exdb/mnist/`. We can reenable this after torchvision dataset is switched to S3 mirror.

```
Mar 04 21:01:04 ======================================================================
Mar 04 21:01:04 ERROR: test_accurracy (__main__.TrainMnist)
Mar 04 21:01:04 ----------------------------------------------------------------------
Mar 04 21:01:04 Traceback (most recent call last):
Mar 04 21:01:04   File "test/test_train_mnist.py", line 186, in test_accurracy
Mar 04 21:01:04     self.assertGreaterEqual(train_mnist(), FLAGS.target_accuracy)
Mar 04 21:01:04   File "test/test_train_mnist.py", line 74, in train_mnist
Mar 04 21:01:04     transforms.Normalize((0.1307,), (0.3081,))]))
Mar 04 21:01:04   File "/var/lib/jenkins/.local/lib/python3.6/site-packages/torchvision/datasets/mnist.py", line 70, in __init__
Mar 04 21:01:04     self.download()
Mar 04 21:01:04   File "/var/lib/jenkins/.local/lib/python3.6/site-packages/torchvision/datasets/mnist.py", line 137, in download
Mar 04 21:01:04     download_and_extract_archive(url, download_root=self.raw_folder, filename=filename, md5=md5)
Mar 04 21:01:04   File "/var/lib/jenkins/.local/lib/python3.6/site-packages/torchvision/datasets/utils.py", line 260, in download_and_extract_archive
Mar 04 21:01:04     download_url(url, download_root, filename, md5)
Mar 04 21:01:04   File "/var/lib/jenkins/.local/lib/python3.6/site-packages/torchvision/datasets/utils.py", line 97, in download_url
Mar 04 21:01:04     raise e
Mar 04 21:01:04   File "/var/lib/jenkins/.local/lib/python3.6/site-packages/torchvision/datasets/utils.py", line 85, in download_url
Mar 04 21:01:04     reporthook=gen_bar_updater()
Mar 04 21:01:04   File "/opt/conda/lib/python3.6/urllib/request.py", line 248, in urlretrieve
Mar 04 21:01:04     with contextlib.closing(urlopen(url, data)) as fp:
Mar 04 21:01:04   File "/opt/conda/lib/python3.6/urllib/request.py", line 223, in urlopen
Mar 04 21:01:04     return opener.open(url, data, timeout)
Mar 04 21:01:04   File "/opt/conda/lib/python3.6/urllib/request.py", line 532, in open
Mar 04 21:01:04     response = meth(req, response)
Mar 04 21:01:04   File "/opt/conda/lib/python3.6/urllib/request.py", line 642, in http_response
Mar 04 21:01:04     'http', request, response, code, msg, hdrs)
Mar 04 21:01:04   File "/opt/conda/lib/python3.6/urllib/request.py", line 570, in error
Mar 04 21:01:04     return self._call_chain(*args)
Mar 04 21:01:04   File "/opt/conda/lib/python3.6/urllib/request.py", line 504, in _call_chain
Mar 04 21:01:04     result = func(*args)
Mar 04 21:01:04   File "/opt/conda/lib/python3.6/urllib/request.py", line 650, in http_error_default
Mar 04 21:01:04     raise HTTPError(req.full_url, code, msg, hdrs, fp)
Mar 04 21:01:04 urllib.error.HTTPError: HTTP Error 403: Forbidden
```

cc @ezyang @gchanan @zou3519 @ailzhang

**Comment by mrshenli** [comment]
Looks like our chrono job does not work for this test.  I am landing #34261.

https://app.circleci.com/jobs/github/pytorch/pytorch/4702077

@ailzhang @seemethere Please enable this test when https://github.com/pytorch/vision/pull/1940 is in. Thanks!

**Comment by mrshenli** [comment]
Fixed by #34316
